Mathlib/Topology/Sober.lean

```diff
@@ -39,10 +39,10 @@ theorem isGenericPoint_def {x : α} {S : Set α} : IsGenericPoint x S ↔ closur
   Iff.rfl
 #align is_generic_point_def isGenericPoint_def
 
-theorem IsGenericPoint.def {x : α} {S : Set α} (h : IsGenericPoint x S) :
+theorem IsGenericPoint.def' {x : α} {S : Set α} (h : IsGenericPoint x S) :
```

Comment:
```suggestion
-- Adaptation note: 2024-03-15
-- Renamed to avoid the reserved name `IsGenericPoint.def`.
theorem IsGenericPoint.def' {x : α} {S : Set α} (h : IsGenericPoint x S) :
```